packages/semantic-release-config/index.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
       "@semantic-release/git",
       {
         assets: ["package.json", "docs/CHANGELOG.md"],
+        "message": "chore(release): ${nextRelease.version} [skip ci]"
```

Comment:
by default this message also includes release notes https://github.com/semantic-release/git